### DISCUSSION TITLE
Web interface not accessible since update.

### DISCUSSION THREAD

**Discussion by Yamanipanuchi** [guydavis/machinaris#349] [Q&A] [answered]
I've been getting the following error since recent update.

![image](https://user-images.githubusercontent.com/4724631/138314819-fe553af1-dff8-4b46-ad20-f7ab1620b0d9.png)

Running on an UnRaid server 6.9.2.

I don't see anyone else having this issue, So it may be isolated to me. Looking for some ideas to look at?

**Comment by guydavis** [comment, marked as answer]
Hi, please provide /mnt/user/appdata/machinaris/machinaris/logs/webui.log which will have the error.

Please follow the [upgrade guide](https://github.com/guydavis/machinaris/wiki/Unraid#how-do-i-update-from-v05x-to-v060-with-fork-support).  Then try this:

1. Stop all Machinaris containers in the Unraid UI | Docker tab.
2. Remove file /mnt/user/appdata/machinaris/machinaris/dbs/machinaris.db holding stale status records.
3. Check for updates for containers.
4. Start up all Machinaris containers
5. Wait about 10-15 minute for all the containers to re-populate active status.

Ideally, the stale records are gone and only active status remains. Please let me know how it goes. If this solves your issue, please mark the Discussion as Answered.

**Reply by Yamanipanuchi** [reply]
[webui.log](https://github.com/guydavis/machinaris/files/7391588/webui.log)

I followed the upgrade guide and made the needed changes and followed what you listed above. Unfortunately no change.

I also did a force update to the container.

Id id nottcie the following in  log  in the docker ui for unraid.


`Welcome to Machinaris!
Only one blockchain allowed per container in Machinaris.
Please remove second value from 'blockchains' variable and restart.
./entrypoint.sh: line 11: return: can only `return' from a function or sourced script
Configuring Chia...
Adding key at path: /root/.chia/mnemonic.txt
Daemon not started yet
Starting daemon
chia_harvester: started
chia_farmer: started
chia_full_node: started
chia_wallet: started
/machinaris/scripts/forks/flax_launch.sh: line 6: cd: /flax-blockchain: No such file or directory
Configuring Flax...
Adding key at path: /root/.chia/mnemonic.txt
/machinaris/scripts/forks/flax_launch.sh: line 30: flax: command not found
/machinaris/scripts/forks/flax_launch.sh: line 39: flax: command not found
/machinaris/scripts/forks/flax_launch.sh: line 39: flax: command not found
/machinaris/scripts/forks/flax_launch.sh: line 39: flax: command not found
/machinaris/scripts/forks/flax_launch.sh: line 50: flax: command not found
/machinaris/scripts/forks/flax_launch.sh: line 57: flax: command not found
Configuring Plotman...
Configuring Chiadog...
cp: cannot stat '/machinaris/config/chia,flaxdog.sample.yaml': No such file or directory
Starting Chiadog...
Starting Machinaris API server...
Starting Machinaris Web server...
Completed startup. Browse to port 8926.`

**Reply by guydavis** [reply]
Hi, please set blockchains=chia. You have blockchains=chia,flax  i think

**Reply by Yamanipanuchi** [reply]
I thought the same thing but,

![image](https://user-images.githubusercontent.com/4724631/138327965-4057f21a-042f-4206-bace-704ac4b95df4.png)

I had to add this variable to the container as it was not there before.

**Reply by Yamanipanuchi** [reply]
I can confirm the farming IS working, just the Web Interface is not.

**Reply by guydavis** [reply]
on my phone, so terse

blockchains was there, go to Advanced View, click "Show more settings", you now have duplicate

**Reply by Yamanipanuchi** [reply]
That was it! Web UI Now working! Thank you!

**Reply by guydavis** [reply]
Great! v0.6 upgrade is a pain. will be much easier in future though. will be adding more forks

fyi, flora fork is NOT compatible with v0.6.0 today, others are fine though
